bindings/pyroot/JupyROOT/helpers/utils.py

```diff
@@ -47,7 +47,7 @@
 <div id="{jsDivId}"
      style="width: {jsCanvasWidth}px; height: {jsCanvasHeight}px">
 </div>
-
+<script src="/static/components/requirejs/require.js?v=951f856e81496aaeec2e71a1c2c0d51f" type="text/javascript" charset="utf-8"></script>
```

Comment:
What is the `v` variable that is being set here? Is it strictly required?

Reply:
I just tried it without the `v`. It still works so I will remove it.